packages/migrate/test/fixtures/project/.prettierrc

```diff
@@ -0,0 +1,20 @@
+{
```

Comment:
Since we are shifting this work into prettier. We need to be sure the migrating project in question has a similar config.